core/jazz_ui/src/config/toastmessages.ts

```diff
@@ -187,5 +187,26 @@ export const toastMessage = {
         "errorIndicationMsg": "There must be at least one user delegated under each category. To add additional users, select \"Add User\" from the options below.",
         "removeAdminRule": "When saved, user will lose all kinds of access to this service. Do you want to proceed?",
         "addAdminRule": "When saved, user will be an administrator to this service. Do you want to proceed?"
+    },
+    "metricsResponse": {
+        "dataNull":"Something went wrong while we were fetching service metrics data!",
+        "error400":"Something went wrong while we were fetching service metrics data!",
+        "error401":"Token expired",
+        "error404":"Something went wrong while we were fetching service metrics data!",
+        "error500":"Something went wrong while we were fetching service metrics data!"
+    },
+    "getDeploymentsResponse": {
+        "dataNull":"Something went wrong while we were fetching service metrics data!",
```

Comment:
Looks incorrect to me (shows metrics data under deployments section?). Is this expected to be shown when there are errors while calling deployments API?

Reply:
Updated the message